qiskit/transpiler/passes/mapping/depth_bounded_mapper.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# Copyright 2019 Andrew M. Childs, Eddie Schoute, Cem M. Unsal
```

Comment:
Is this line compatible with the line 5?

Reply:
I don't know. Any suggestions?

Reply:
This is fine, it's providing attribution to the original authors and outlining this is a derivative work of the original, and the licenses are compatible (because they're both Apache). It's a bit redundant since the license text is almost the same and both are saying its apache. Normally I would put the copyright lines next to each other and only have a single apache header. But, since the header text differs it's probably better to err on the side of caution for such things and put both.